include/yaramod/types/token_stream.h

```diff
@@ -136,6 +142,7 @@ class TokenStream
 private:
 	std::list<Token> _tokens; ///< All tokens off the rule
 	bool formatted = false; ///< The flag is set once autoformat has been called
+	std::string new_line_style = "\n"; ///< The character used for line endings: usually '\n' on Unix or '\r' on MacOs or '\r\n' on Windows
```

Comment:
Since these are private attributes, I think they should be prefixed with `_` to match conventions used everywhere throughout `yaramod`.